docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -113,6 +114,12 @@ Start by creating a `pom.xml` for your Java application:
       by replacing the repositories in the example POM above with a repository with this
       URL instead. Also update `ksqldb.version` to be a CP version, such as `{{ site.cprelease }}`, instead.
 
+!!! note
+      The `with-dependencies` artifact was introduced in ksqlDB version 0.29 and CP version 7.4.0.  This jar includes  
```

Comment:
I'm having problems groking this:

> you can depend directly on the client by removing that classifier from the dependency block.

Are you saying that if you have vertx pinned in your own project, and you don't want to take it from `ksqldb-api-client` then you can _remove_ the `with-dependencies` classifier? That makes sense, but for some reason I'm struggling with the english.

Reply:
Yeah, English is hard. 

I'm trying to suggest that folks could assemble their own classpath by depending on `io.confluent.ksql:ksqldb-api-client` directly.  

Victoria is basically asking the same question (I think).  Suggestions for how to clarify what I'm saying here?

Reply:
I think an example would go a long way (e.g. if you want to do X do Y)